build_tools/azure/posix.yml

```diff
@@ -15,11 +15,22 @@ jobs:
     TEST_DIR: '$(Agent.WorkFolder)/tmp_folder'
     VIRTUALENV: 'testvenv'
     JUNITXML: 'test-data.xml'
-    PYTEST_VERSION: '5.2.1'
-    PYTEST_XDIST: 'true'
     OMP_NUM_THREADS: '2'
     OPENBLAS_NUM_THREADS: '2'
     SKLEARN_SKIP_NETWORK_TESTS: '1'
+    NUMPY_VERSION: 'latest'
+    SCIPY_VERSION: 'latest'
+    CYTHON_VERSION: 'latest'
+    JOBLIB_VERSION: 'latest'
+    PANDAS_VERSION: 'latest'
+    PYAMG_VERSION: 'latest'
+    PILLOW_VERSION: 'latest'
+    MATPLOTLIB_VERSION: 'latest'
+    PYTEST_VERSION: 'latest'
+    PYTEST_XDIST_VERSION: 'latest'
+    THREADPOOLCTL_VERSION: 'latest'
+    COVERAGE: 'false'
```

Comment:
since we have bits of code which only run if a recent version of a dependency is installed, shouldn't `coverage` be `true` here?